serial_unix.go

```diff
@@ -511,5 +510,5 @@ func (port *unixPort) enableRS485(config *RS485Config) error {
 		rs485.flags |= rs485RXDuringTX
 	}
 
-	return unix.IoctlSetInt(port.handle, rs485Tiocs, int(uintptr(unsafe.Pointer(&rs485))))
+	return unix.IoctlSetInt(port.handle, unix.TIOCSRS485, int(uintptr(unsafe.Pointer(&rs485))))
```

Comment:
Nice find!

Reply:
Tragically not though - it causes build issues on other platforms. The constant is only defined for Linux builds. Possibly that's an argument (since we're doing a Linux ioctl here) for moving this code out of `serial_unix.go` and into `serial_linux.go`?